### ISSUE TITLE
View of specific message

### ISSUE THREAD

**Issue by 33mhz** [sunya9/beta#179]
**Is your feature request related to a problem? Please describe.**
I'd like to be able to open a page to a specific message, like we can currently pull up a specific post.

**Describe the solution you'd like**
I'd like a URL to specify a message location, like `/@{username}/posts/{post_id}` and `/posts/{post_id}` for posts.

**Describe alternatives you've considered**
Adding an `id` to each item in a list and then specifying `/messages/{room_id}#{message_id}` is a start, though of course only works if the message is initially loaded. (and the header bar blocks the targeted message).

**Comment by sunya9** [comment]
Sounds good. But that is inconveniences in case of the target messages does not be included in initial loading.
If you would like to reference target messages directly from URL, it must be included at the stage of SSR but the server cannot find a hash.
For that, I think that it is better to use query string instead of the hash, like `/messages/{room_id}?message={message_id}`.
After, it will realize to use paginate.

**Comment by 33mhz** [comment]
Yeah, that's why the alternative isn't a good option. I think `/messages/{room_id}/{message_id}` would be fine, maybe confusing to look at though.

**Comment by 33mhz** [comment]
Another option for this feature would be to simply load `/message/{message_id}` using the [GET /channels/{channel_id}/messages/{message_id}/thread](https://pnut.io/docs/api/resources/messages/lookup#get-channels-id-messages-id-thread) endpoint. This would be paginated identically to `/posts/{post_id}` threads are.
